### PR TITLE
Rework sidebar and fix bugs

### PR DESCRIPTION
**Pull request by takkaria**

### CODE REVIEW

**Comment by idrysdale** [comment]
I'm having a bit of difficultly testing this, sorry!

I tried adding a new scenario, but since it duplicates *Master* or an *existing scenario*, the kgCO² value is already set. How did you manage to make one?

In trying to test it, I noticed when I created a whole new assessment it resulted in *Master* initially having 'NaN kgCO²'. From what I can tell this is set elsewhere within `view.html` - do you think it's worth catching that instance of writing it out in this PR, too?

**Comment by takkaria** [comment]
OK, so I've worked on this like you suggested - was a bit of a wild goose chase in the end.

To reproduce the original issue, I did the following:

1. Create a new assessment
2. Create a new scenario, copied from 'master'
3. Check the console - there's the error `TypeError: project[s].kgco2perm2 is null`
4. If you hit refresh, you can see the new scenario wasn't saved

File: mhep/mhep/dev/templates/dev/view.html [hunk 1]
```diff
@@ -426,18 +435,14 @@ <h3>This scenario is locked!</h3>
         $(window).scrollTop(650);
     });
     $("#openbem").on('click', ".block-header", function () {
+        const project_id = this.closest("[scenario]").getAttribute('scenario');
```
Comment (idrysdale) [inline mhep/mhep/dev/templates/dev/view.html]: When I click 'Create new scenario' now, I'm getting: `TypeError: this.closest(...) is null` in the console.

**Comment by idrysdale** [comment]
Oh well done! Sorry that spiraled there! I thought it was a UI thing you were fixing, but if I understand it, it was a console error after all.

I went to test it, but got the error commented above ☝️

**Comment by idrysdale** [comment]
This looks great! Nice one! 👏👏👏

**Pull request merged**